bootstrap/css/index.css

```diff
@@ -0,0 +1,232 @@
+    box-sizing: border-box;
+}
+
+.top-story-fig h2{
+  font-size: 1.375rem;
+  font-weight: 700;
+}
+
+.top-story-fig h3{
+  font-size: 1.25rem;
+  font-weight: 700;
+}
+
+.top-story-fig h2 a{
+  color:#000;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  z-index: 2;
+}
+
+.col {
+  height: 172px;
+}
+.col-md-5 h3, .col-12 h3 {
+  font-size:1.0rem;
+  color:#000;
+  font-weight: 700;
+}
+.col-md-5 p,  .col-12 p{
+  font-size: 0.8125rem;
+}
+.cover{
+    object-fit: cover;
+}
+
+.head-nav span:last-child{
+  background-color: white;
+  color: red;
+  cursor: pointer;
+  font-size: 1.125rem;
+  font-weight: 700;
+}
+
+.head-nav span:first-child{
+    cursor: pointer;
+    font-size: 1.125rem;
+    font-weight: 700;
+}
+.navigation{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border-bottom:
+}
+
+.date{
+  font-size: 1rem;
+}
+
+.container-fluid.navigation.p-3.border-bottom li{
+  color: black;
+  font-weight: 700;
+  font-size: 1.3125rem;
+  height: 24px;
+  list-style: none;
+}
+.img_holder{
+  width: 63px;
+  height: 63px;
+  border-radius: 50%;
+}
+.img_holder img{
+  width: auto;
+  height: 63px;
+}
+.latest-news-image_holder {
+  width: 135px;
+  height: 100px;
+}
+.latest-news-image_holder img{
+  width: 135px;
+  height: auto;
+}
+.opinion .info div, .latest-news .info div{
+    font-weight: bold;
+    font-stretch: normal;
+    font-size: 17px;
+    line-height: 21px;
+    font-family: "Roboto Condensed", sans-serif;
+    color: #000;
+}
+
+.opinion .info h4{
+     font-size: 14px;
+    line-height: 21px;
+    color: #555;
+    text-transform: uppercase;
+    margin-top: 4px;
+}
+.special-Edition{
+  height: 400px;
+  width:;
+  text-align: center;
+}
+.special-Edition-wrapper{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  margin: 0px;
+}
+
+.col-md-4 .the-other-section-title {
+  margin-top: 10px;
+  margin-bottom: 25px;
+ }
+
+@media (min-width: 768px){
+  .col-md-4 .the-other-section-title{
+    font-size: 1.125rem;
+    line-height: 18px;
+  }
+}
+
+@media (min-width: 768px){
+  .col-md-12.d-flex.flex-column.align-self-center img
+  {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+}
+
+@media (min-width: 576px){
+  .the-magazine-image-holder .px-3.py-1
+  {
+   color: red;
+   background-color: rgba(255,255,255,.9);
+   position: absolute;
+   bottom: 160px;
+   left: 70px;
+   z-index: 1;
+   font-weight: 700;
+  }
+}
+
+@media (min-width: 992px){
+    .top
+  {
+    position:relative;
+    z-index: 1;
+    top: 180px;
+  }
+
+  .content.footer
+  {
+    position:relative;
+    z-index: 1;
+    top: 180px;
+  }
+}
+@media (max-width: 575px){
+    .content.footer
+  {
+    position:relative;
+    z-index: 1;
+    top: 20px;
+  }
+}
+
+
+
+
+.page-footer .col .copyright {
+    font-size: 12px;
+    font-family: Arial,Helvetica,sans-serif;
```

Comment:
Opinion: Could also use font shorthand for this an above line (not required)
Could do same for other similar cases